### ISSUE TITLE
Allow repeating `@Path` annotations

### ISSUE THREAD

**Issue by trustin** [line/armeria#1822]
A user wants to specify multiple path pattens for a single service method in an annotated service. For example:

```java
@Get
@Post
@Path("/v1/some_api")
@Path("/v2/some_api")
```

Alternatively, we could consider allowing repeating `@METHOD` annotations, e.g.

```java
@Post("/v1/some_api")
@Post("/v2/some_api")
```

**Comment by jrhee17** [comment]
mind if I work on this? :D

**Comment by minwoox** [comment]
👍

**Comment by jrhee17** [comment]
I feel like the first proposal and second proposal both are reasonable and can be worked on in separate iterations. I'll start with working on the first proposal (allowing multiple paths, but not multiple `@METHOD` annotations)

**Comment by trustin** [comment]
OK for supporting both patterns.

**Comment by minwoox** [comment]
Closed by #1870
